README.md

```diff
@@ -36,7 +36,15 @@ bosh -e YOUR_ENV -d app-autoscaler \
      -v cf_admin_password=<cf admin password> \
      -v skip_ssl_validation=true
 ```
-
+To consume fewer bosh vms with app-autoscaler-deployment-fewer.yml
```

Comment:
To deploy app-autoscaler with less VMs, use `app-autoscaler-deployment-fewer.yml`